src/actions/drafts.js

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable import/prefer-default-export */
+import axios from 'axios';
+import * as types from '../actionTypes/drafts';
+
+const API = process.env.REACT_APP_API;
+
+
+// Action types for saving articles as drafts
+const createDraft = (isSaving) => ({
+  type: types.CREATE_DRAFT,
+  payload: isSaving,
+});
+
+const createDraftSuccess = (newDraft) => ({
+  type: types.CREATE_DRAFT_SUCCESS,
+  payload: newDraft,
+});
+
+const createDraftError = (error) => ({
+  type: types.CREATE_DRAFT_ERROR,
+  payload: error,
+});
+
+// Draft action
+export const draftArticle = (articleData) => (dispatch) => {
+  dispatch(createDraft(true));
+  const { token } = localStorage;
+  return axios.post(
+    `${API}/api/articles`, articleData, {
+      headers: {
```

Comment:
This endpoint is it for saving draft or for creating an article?

Reply:
Yes. @CEOehis The same endpoint handles both actions